handsontable/src/plugins/columnSorting/__tests__/columnSorting.spec.js

```diff
@@ -1051,7 +1051,7 @@ describe('ColumnSorting', () => {
           {},
           {
             type: 'date',
-            dateFormat: 'mm/dd/yy'
+            dateFormat: 'MM/DD/YYYY'
```

Comment:
This file also uses the old notation `dateFormat: 'mm/dd/yy'` on lines 294, 1532 (and also in 2 places in `multiColumnSorting.spec.js`). 

Why were other lines not affected?

Reply:
Those remaining examples seem to not be sorted by the column containing the dates, so the `dateFormat` is pretty much redundant, I'll change it, though.